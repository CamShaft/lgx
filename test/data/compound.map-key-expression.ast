#{
  type => map,
  line => 1,
  children => [
    #{
      type => tuple,
      children => [
        #{
          type => literal,
          value => did_it_work,
          line => 2
        },
        #{
          type => literal,
          value => it_worked,
          line => 2
        }
      ]
    }
  ]
}.