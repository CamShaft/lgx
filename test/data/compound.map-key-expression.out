#{
  did_it_work => it_worked
}.